assignments/array-methods.js

```diff
@@ -55,29 +55,60 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
-let fullName = [];
+let fullName = runners.map(person => `${person.first_name} ${person.last_name}`);
 console.log(fullName);
```

Comment:
.map() does work here, but you were supposed to use forEach :) 

let fullName = [];
let conferenceNames = runners.forEach(function(i){
  fullName.push(i.first_name + " " + i.last_name);
});
console.log(fullName);